packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -45,8 +45,8 @@ export const FormattedCryptoAmount = ({
     signValue,
     disableHiddenPlaceholder,
     isRawString,
+    justifyContent = 'flex-end',
     'data-testid': dataTest,
-    className,
```

Comment:
all the occurences of `styled(FormattedCryptoAmount)` will stop working

Reply:
😳

Reply:
It's bacl